spec/42sh/globbing/bracket/multi/001-range-and-char/stdin

```diff
@@ -0,0 +1 @@
+echo ${GLOBAL_TMP_DIRECTORY}/test_globbing/[a-z-2]
```

Comment:
we should never use "echo" but "./write_on_stdout" instead, or any other support file ;-)